TestProjects/NotificationSamples/Assets/Scripts/Helpers/GameObjectReferences.cs

```diff
@@ -0,0 +1,10 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+public class GameObjectReferences : MonoBehaviour
+{
+    public Text logsText;
+    public Transform buttonScrollViewContent;
+    public Transform buttonGroupTemplate;
+    public ScrollRect logsScrollRect;
```

Comment:
Public fields should start with Capital according to our coding covention.

Reply:
fixed these, and some others I found, hope I found them all